scripts/fire.shader

```diff
@@ -1,16 +1,20 @@
-gfx/weapons/flamer/fire/fire
+gfx/weapons/firebomb/fire
 {
-	qer_editorImage gfx/weapons/flamer/fire/fire_p
-//	q3map_surfacelight 1000
+	qer_editorImage gfx/weapons/firebomb/fire/fire-01
+
 	surfaceparm nomarks
 	surfaceparm nolightmap
 	surfaceparm trans
+
 	polygonOffset
 	cull none
 	sort nearest
-	imageMinDimension 128
+
+	imageMinDimension 64
+
 	{
-		animMap 12 gfx/weapons/flamer/fire/fire1 gfx/weapons/flamer/fire/fire2 gfx/weapons/flamer/fire/fire3 gfx/weapons/flamer/fire/fire4 gfx/weapons/flamer/fire/fire5 gfx/weapons/flamer/fire/fire6 gfx/weapons/flamer/fire/fire7 gfx/weapons/flamer/fire/fire8
-		blendFunc GL_ONE GL_ONE
+		animMap 20 gfx/weapons/firebomb/fire/fire-01 gfx/weapons/firebomb/fire/fire-02 gfx/weapons/firebomb/fire/fire-03 gfx/weapons/firebomb/fire/fire-04 gfx/weapons/firebomb/fire/fire-05 gfx/weapons/firebomb/fire/fire-06 gfx/weapons/firebomb/fire/fire-07 gfx/weapons/firebomb/fire/fire-08 gfx/weapons/firebomb/fire/fire-09 gfx/weapons/firebomb/fire/fire-10 gfx/weapons/firebomb/fire/fire-11 gfx/weapons/firebomb/fire/fire-12 gfx/weapons/firebomb/fire/fire-13 gfx/weapons/firebomb/fire/fire-14 gfx/weapons/firebomb/fire/fire-15 gfx/weapons/firebomb/fire/fire-16 gfx/weapons/firebomb/fire/fire-17 gfx/weapons/firebomb/fire/fire-18 gfx/weapons/firebomb/fire/fire-19 gfx/weapons/firebomb/fire/fire-20 gfx/weapons/firebomb/fire/fire-21 gfx/weapons/firebomb/fire/fire-22 gfx/weapons/firebomb/fire/fire-23 gfx/weapons/firebomb/fire/fire-24 gfx/weapons/firebomb/fire/fire-25
+
+		blendFunc add
```

Comment:
Is this the part that supposedly fixes a bug about showing through walls?

Reply:
no, `blendFunc add` is just an alias for `blandFunc GL_ONE GL_ONE`:

https://github.com/DaemonEngine/Daemon/blob/master/src/engine/renderer/tr_shader.cpp#L2429-L2445